web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -121,7 +121,7 @@
         // Retrieve site information
         // TODO: Add INSPIRE, harvester, ... information
         var catInfo = promiseStart.then(function(value) {
-          var url = $scope.url + 'info?_content_type=json&type=site&type=auth';
+          var url = $scope.url + 'info?_content_type=json&type=site&type=auth&type=config';
```

Comment:
I use that in ```mdactionmenu.html```: 

```
 data-ng-class="(md.isPublished() || 
(info['metadata.workflow.allowPublishInvalidMd'] === 'true') || ...
```

As the ```info``` is available in the scope defined in ```CatController```, thats the main reason to extend it. I can't use ```gnConfig``` in ```mdactionmenu.html```, seem not in the scope. For sure, I can add a method in ```CatController``` like this and use it in the html code:

```
$scope.allowPublishInvalidMd = function() {
        return gnConfig['metadata.workflow.allowPublishInvalidMd'];
      };
```

Or there's any other preferred way to get access from html files of the config?